### ISSUE TITLE
please help,i cannot recovery my laverna data.

### ISSUE THREAD

**Issue by clamever** [Laverna/laverna#884]
my computer got collapse in the day before yesterday(the computer system is ubuntu).because i am the new comer to the laverna. so i did not know i should export settings file to backup.i just knew that after my computer got collapse.fortunately,i can recovery the disk data.and i copy the ~/.config/laverna and the laverna program directory to the new computer.(~/.config/laverna is the data directory of the laverna ,right?there is really few documents about laverna).

then i start the laverna program in my new computer,it seems worked because i can decrypted my laverna note with my password.but i only found few notes in my laverna notebook,it means ,few notes recovery but most did not.i always keep dropbox sync in my old computer.it seems my settings already recovery in my new laverna,why it did not resync the full data?it is really weired.

**Comment by clamever** [comment]
i am Chinese,sorry for my rough English.

**Comment by KenrySanchez** [comment]
I don't understand at well. Don't you synchronize with your Dropbox application?

**Comment by clamever** [comment]
yes , i can sure i keep the synchronization with my Dropbox application in the before.

**Comment by clamever** [comment]
can you tell me is the ~/.config/laverna data directory of laverna?

**Comment by KenrySanchez** [comment]
I'm not really sure if it's the directory but it's rare that you can't synchronize your notes. I'm searching it.

**Comment by clamever** [comment]
thanks,the documents of the laverna is really few.reallty hope there would be more detailed docs about the laverna.i wanted to read the source code to know deeply but i always did not have time to do that.

**Comment by wwebfor** [comment]
Yes, the data directory on Linux is ~/.config/laverna

**Comment by upsatish** [comment]
@clamever
If you have done a drop box sync, wouldn't the notes (encrypted if encryption was enabled) be stored in Dropbox\Apps\Laverna and not in the data directory on disk?
In that case you would only need your settings file to reuse the Laverna notes database?
